akka-docs/src/main/paradox/scala/general/actor-systems.md

```diff
@@ -107,12 +107,209 @@ which is a single point of contention if over-used.
 In some cases it is unavoidable to do blocking operations, i.e. to put a thread
 to sleep for an indeterminate time, waiting for an external event to occur.
 Examples are legacy RDBMS drivers or messaging APIs, and the underlying reason
-is typically that (network) I/O occurs under the covers. When facing this, you
+is typically that (network) I/O occurs under the covers. 
```

Comment:
Shall we move this whole section into the `dispatcher.md` documentation instead? I always have trouble finding it (ending up searching). This "general" part of the documentation is more about theoretical concepts and this very practical topic doesn't belong here.

We could keep a bullet about it in above Actor Best Practices list with a link to the details.

wdyt?